src/table.ts

```diff
@@ -1414,7 +1414,7 @@ class Table extends common.ServiceObject {
             },
           } as {},
           request: {
-            uri: `https://${this.bigQuery.apiEndpoint}/upload/bigquery/v2/projects/${this.bigQuery.projectId}/jobs`,
+            uri: `${this.bigQuery.apiEndpoint}/upload/bigquery/v2/projects/${this.bigQuery.projectId}/jobs`,
```

Comment:
It would probably be better if we changed the test, instead:

```diff
      it('should pass the correct request uri', done => {
        makeWritableStreamOverride = (
          stream: stream.Stream,
          options: MakeWritableStreamOptions
        ) => {
          const uri =
-           'https://bigquery.googleapis.com/upload/bigquery/v2/projects/' +
+           table.bigQuery.apiEndpoint + '/upload/bigquery/v2/projects/' +
            table.bigQuery.projectId +
            '/jobs';
          assert.strictEqual(options.request.uri, uri);
          done();
        };

        table.createWriteStream_().emit('writing');
      });
```

Reply:
I snuck a commit in to do this so we can merge and release in 5.3.0 along with another feature.